### PR TITLE
add: change language by url parameter

### PR DESCRIPTION
**Pull request by fcoronelmakingsense**
- Add query-string library to manage and parse query string from location
https://github.com/sindresorhus/query-string

- Change languague by url `lang` parameter

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -26,21 +27,31 @@ class App extends Component {
 
   componentDidMount() {
     this.sessionManager.initialize(this.updateSession);
+    this.setUrlLanguage(this.state.dopplerSession);
```
Comment (andresmoschini) [inline src/App.js]: Parameter is not necessary, right?

```suggestion
    this.setUrlLanguage();
```

**Comment by andresmoschini** [comment]
Nice! And, only to know, how does it work with authenticated pages?

**Comment by fcoronelmakingsense** [comment]
> Nice! And, only to know, how does it work with authenticated pages?

for now, use user language in this cases

File: src/App.js (hunk 1 above)
Comment (fcoronelmakingsense) [reply src/App.js]: Thanks! Fixed

**Pull request merged**